ckanext/canada/tables/reclassification.yaml

```diff
@@ -110,7 +110,7 @@ resources:
     format_type: Controlled List
     validation: This field must not be empty
     excel_full_text_choices: True
-    form_choices_prefix_code: True
+    form_full_text_choices: True
```

Comment:
prefix_code is a better name because the form always shows the full text. For excel the default is to only show the code

Reply:
:shrug: but if that's how we've already implemented it I guess it's fine

Reply:
@wardi okay I changed it: `form_full_text_choices` -> `form_choices_prefix_code`. Updated usages in NAP5 schema as well.